appcenter-data/ios/AppCenterReactNativeData/AppCenterReactNativeData.m

```diff
@@ -96,6 +96,18 @@ + (void)register {
     }
 }
 
+RCT_EXPORT_METHOD(isEnabled:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject) {
+    resolve([NSNumber numberWithBool:[MSData isEnabled]]);
```

Comment:
```suggestion
    resolve(@([MSData isEnabled]));
```
This should work here

Reply:
I believe `numberWithBool` is copied from other module's `isEnabled` method (e.g. https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-analytics/ios/AppCenterReactNativeAnalytics/AppCenterReactNativeAnalytics.m#L62).

Then we can fix all occurrences to improve readability.

Reply:
Fixed all occurrences. [https://github.com/microsoft/appcenter-sdk-react-native/pull/641/commits/7e0f79703467fc8def69a9622d710aa749025703](https://github.com/microsoft/appcenter-sdk-react-native/pull/641/commits/7e0f79703467fc8def69a9622d710aa749025703)